kafka/source/pkg/reconciler/source/resources/receive_adapter.go

```diff
@@ -57,7 +57,7 @@ func MakeReceiveAdapter(args *ReceiveAdapterArgs) *v1.Deployment {
 		Name:  "KAFKA_NET_TLS_ENABLE",
 		Value: strconv.FormatBool(args.Source.Spec.Net.TLS.Enable),
 	}, {
-		Name:  "SINK_URI",
+		Name:  "K_SINK",
```

Comment:
that is the SinkBinding feature ... 

Do we no longer need the new ?   or was that all changed ?

Reply:
I can't really say why this env changed, i just switched from adapter to adapter/v2 and this happened :)

Reply:
right -- just to elaborate on that, the `K_SINK` is a var in the `adapter.EnvConfig` and it gets resolved to the sink, so that's why it still properly outputs in the logging.